.github/workflows/ci.yml

```diff
@@ -23,11 +23,10 @@ jobs:
 
     - name: Helm Template
       run: |
-       if helm template .. --values ../env/values-prod.yaml | grep -E -i "(error:|failed|Error:)" ; then
+        if helm template ../../. -f ../../env/values-prod.yaml 2>&1 | grep -qi "Error"; then
```

Comment:
Still can't find the right way to check helm template.
Also, the relative path looks bad. How can I refer to root here?

Reply:
You can't really refer to root, you can put a step before that will change the current directory you're in to the helm directory that way the rest of the flow doesn't need to include all these relative paths

Reply:
I think if you have a step with this command           helm template --dry-run  --atomic --wait --timeout ${{ inputs.TIMEOUT }} it should result in failing the action if this fails